src/test/integration/integrationtests.py

```diff
@@ -171,17 +171,17 @@ def test_bad_post_job(self):
         positive_fields = ["hourlyRate", "hoursPerPay", "assignmentSalary",
                            "annualSalary", "paysPerYear"]
         for field in positive_fields:
-            self.validate_bad_job_post(test_job, field, "-1",
+            self.validate_bad_job_post(test_job, field, -1,
```

Comment:
I did some testing to confirm that the api doesn't have a problem with these values being strings but I changed it to a number to follow the documentation